src/context-selection/org-unit-selector-bar-item/use-org-unit-paths-by-name.js

```diff
@@ -11,19 +11,14 @@ export default function useOrgUnitPathsByName(searchTerm) {
             },
         },
     ]
-    const {
-        isIdle,
-        isLoading: loading,
-        error,
-        data,
-    } = useQuery(queryKey, {
+    const { isIdle, isFetching, error, data } = useQuery(queryKey, {
         enabled: !!searchTerm,
         select: (data) => data.organisationUnits.map(({ path }) => path),
     })
 
     return {
         called: !isIdle,
-        loading,
+        loading: isFetching,
```

Comment:
"isLoading" behaviour changed in `react-query@4`. Now `isLoading` is `true` if the query is `disabled`.